api/soap/mc_issue_api.php

```diff
@@ -1010,9 +1010,12 @@ function mc_issue_update( $p_username, $p_password, $p_issue_id, stdClass $p_iss
 	$t_category = isset( $p_issue['category'] ) ? $p_issue['category'] : null;
 	$t_category_id = mci_get_category_id( $t_category, $t_project_id );
 
-	$t_version_id = isset( $p_issue['version'] ) ? mci_get_version_id( $p_issue['version'], $t_project_id, 'version' ) : 0;
-	$t_fixed_in_version_id = isset( $p_issue['fixed_in_version'] ) ? mci_get_version_id( $p_issue['fixed_in_version'], $t_project_id, 'fixed_in_version' ) : 0;
-	$t_target_version_id = isset( $p_issue['target_version'] ) ? mci_get_version_id( $p_issue['target_version'], $t_project_id, 'target_version' ) : 0;
+	$fn_get_version_id = function( $p_version, int $p_project_id, string $p_field ): int {
+		return isset( $p_version ) ? mci_get_version_id( $p_version, $p_project_id, $p_field ) : 0;
```

Comment:
To empty a version field set its id to 0. I added a note to the bug with more details.

Reply:
I replied in the tracker https://mantisbt.org/bugs/view.php?id=25407#c68848